docs/build_yaml_format.md

```diff
@@ -8,12 +8,13 @@ to see examples or goal oriented docs you may want to look at the
 
 ## BuildConfig
 
-key                   | value                                                                      | default
---------------------- | -------------------------------------------------------------------------- | -------
-targets               | Map<String, [BuildTarget](#buildtarget)>                                   | a single target with the same name as the package
-builders              | Map<String, [BuilderDefinition](#builderdefinition)>                       | empty
-post_process_builders | Map<String, [PostProcessBuilderDefinition](#postprocessbuilderdefinition)> | empty
-global_options        | Map<String, [GlobalBuilderOptions](#globalBuilderOptions)>                 | empty
+key                      | value                                                                      | default
```

Comment:
I should mention that this markdown document has other tables whose columns are all aligned. Since you have an internal formatting tool, maybe you could quickly run that over this file so that I don't have to manually fix them?

Reply:
The toll I was using is internal. I went ahead and pushed a commit which runs it on this file, but looks like most of the formatting had already been handled. Sorry for the difficulty.